src/renderer/features/fellowship-voting/index.tsx

```diff
@@ -50,9 +69,33 @@ fellowshipVotingFeature.inject(taskVotingActionSlot, ({ referendumId }) => {
   };
 
   return (
-    <Box direction="row" gap={3}>
-      <FilledIconButton variant="negative" icon="thumbDown" disabled={disabled} onClick={nay} />
-      <FilledIconButton variant="positive" icon="thumbUp" disabled={disabled} onClick={aye} />
+    <Box gap={1}>
+      <Box direction="row" gap={0.5}>
+        <FilledIconButton
+          variant="negative"
+          icon="thumbDown"
+          disabled={disabled}
+          voted={referendumVote?.decision === 'Nay'}
+          checked={nonNullable(transaction) && !transaction.args.aye}
```

Comment:
Why it cannot be like this?
```suggestion
          checked={nonNullable(transaction) && transaction.args.nay}
```

Reply:
We don't have `nay` there only `transaction.args.aye: true | false`